2021/05.md

```diff
@@ -107,9 +107,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
 
 <!-- Be specific! Provide a full name, date and time range that they will or will not be available, and which sessions they are trying to prioritize. Satisfaction not guaranteed, but more information is useful. Conflicting constraints honored on a first-come, first served basis. -->
 1. Frank Yung-Fong Tang is NOT available in the morning of Wednesday May 26th.
-1. Mathias Bynens and Markus Scherer are presenting the RegExp Unicode proposal.
-   1. Markus is available Tue 10-15 PT, Wed 12-15 PT.
-   1. Mathias wrote that he is only available between 12:00–13:00 PST on Wednesday May 26th. Double-check?
+1. Mathias Bynens and Markus Scherer are presenting the RegExp Unicode proposal, and are available between 13:00–14:00 PST on Wednesday May 26th.
```

Comment:
wfm but note that @gesa has taken these constraints into the schedule doc already: https://hackmd.io/@tc39-chairs/S1gavoSK_ -- and it also looks like a number of people want the Realms discussion to be at 13-14 PT (see Aki’s note at the end of that other doc)